core/rfb.js

```diff
@@ -1971,7 +1971,7 @@ RFB.encodingHandlers = {
         return true;
     },
 
-    display_tight: function (isTightPNG) {
+    TIGHT: function () {
```

Comment:
Does the server not send the TIGHT_PNG encoding explicitly?  If so, we should probably have comments on which encodings are just for advertising capabilities, and which are for actually being sent by the server.

Reply:
Yup. That's why I tagged it as a pseudo-encoding and not a "real" encoding. It's similar to how the quality "encodings" turn on JPEG in Tight.

Reply:
ack, alright.  Cool